### PR TITLE
Updates README for react package to reflect changes to development flow.

### PR DESCRIPTION
**Pull request by smurrayatwork**

### CODE REVIEW

File: react/README.md [hunk 1]
```diff
@@ -11,34 +11,58 @@ Mayflower-react is a React component library under the [enterprise design system
...
-
-Please note that [@massds/mayflower-react][npm] comes with prebuilt css, svg and other image files. If you are configuring your own Webpack, you might need to add the following loaders to handle these files:
-``npm install --save-dev css-loader svg-inline-loader file-loader``
-
-You can also kickstart your React project with Mayflower-react referring to the [Mayflower React Starter][react-starter] built with [Create React App](https://reactjs.org/docs/create-a-new-react-app.html#create-react-app).
-
-For a guide and information on the components included in mayflower-react and their functionality, visit our [Mayflower React Storybook][react-storybook]. Click on the Info and Knobs tabs for component prop types, details and options.
+1. Install mayflower-react and mayflower-assets into your project dependency:
+```javascript
+npm i @massds/mayflower-react @massds/mayflower-assets --save
```
Comment (clairesunstudio) [inline react/README.md]: let's also make `@massds/mayflower-assets` a peer dependency in `@massds/mayflower-react`, if we are telling people to install them separately

File: react/README.md [hunk 2]
```diff
@@ -11,34 +11,58 @@ Mayflower-react is a React component library under the [enterprise design system
...
+```
+2. Import components into your App:
+```javascript
+import { Header, Footer, Button } from '@massds/mayflower-react';
+// You can also directly point to each component's path:
+import Header from '@massds/mayflower-react/dist/Header';
```
Comment (clairesunstudio) [inline react/README.md]:
```suggestion
import Header from '@massds/mayflower-react/dist/Header/index.mjs';
```

Comment (smurrayatwork) [reply react/README.md]: ES Modules is explained further down. This change shouldn't happen.

File: react/README.md [hunk 3]
```diff
@@ -11,34 +11,58 @@ Mayflower-react is a React component library under the [enterprise design system
...
-Please note that we are currently relying on symlink for pulling in shared assets into react source code. For Windows users, you need to manually create the symlink before running `npm start`.
-e.g. In cmd prompt:
-- $ `cd src` - make sure you are in the `src` folder in `mayflower/react`
-- $ `rm assets`
-- $ `ln -s ../../assets assets`
 
 #### System Requirements
 
-- node.js, currently standardized on version 10.15.1
+- node.js, currently standardized on version 12.18.1
```
Comment (clairesunstudio) [inline react/README.md]:
```suggestion
- node.js, currently standardized on version 12.18.0
```
see .nvmrc

File: react/README.md (hunk 2 above)
Comment (clairesunstudio) [reply react/README.md]: hmm ok, Can you move this section https://github.com/massgov/mayflower/pull/1109/files#diff-4f8f09a7b7e325026d10475db1bed44bR43-R53  up here to explain the imports based on project ES5/6 settings

**Pull request merged**